proof/crefine/AARCH64/Tcb_C.thy

```diff
@@ -4505,6 +4505,104 @@ lemma decodeSetTLSBase_ccorres:
   apply (auto simp: unat_eq_0 le_max_word_ucast_id)+
   done
 
+lemma scast_seL4_TCBFlag_fpuDisabled[simp]:
+  "scast seL4_TCBFlag_fpuDisabled = tcbFlagToWord (ArchFlag FpuDisabled)"
+  by (clarsimp simp: seL4_TCBFlag_fpuDisabled_def tcbFlagToWord_def archTcbFlagToWord_def)
+
+lemma invokeTCB_SetFlags_ccorres:
+  notes hoare_weak_lift_imp [wp]
+  shows
+  "ccorres (cintr \<currency> (\<lambda>rv rv'. rv = [])) (liftxf errstate id (K ()) ret__unsigned_long_')
+   (invs' and tcb_at' tcb)
+   ({s. thread_' s = tcb_ptr_to_ctcb_ptr tcb}
+         \<inter> {s. clear_' s = clears}
+         \<inter> {s. set_' s = sets}) []
```

Comment:
indentation is wrong, the intersection belongs to the `(` not to the `{s.`
I would also prefer if you used the modern variable binding style (with acutes and {| |} brackets).